walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```

Comment:
I don't see #getRandomQuoteUsingAsyncAwait in this PR - please include the function as well.

Reply:
This index.html is located in the week-3-server random-quote directory. I had to change it around to see the differences in how suggested ways to write the fetch() method. The getRandomQuoteUsingAsyncAwait() method is located in that directory's script.js. No changes were made to that specific script.js.